doc/source/whatsnew/v2.2.2.rst

```diff
@@ -30,7 +30,7 @@ Bug fixes
 
 Other
 ~~~~~
--
+- Deprecated using ``epoch`` date format in :meth:`DataFrame.to_json` and :meth:`Series.to_json`, use ``iso`` instead.
```

Comment:
Should I go back and deprecate using the `date_format` parameter?
My first thought was that if I did that, then using the `iso` format will throw a deprecation warning, which I don't think is quite right because `epoch` will be deprecated and it's good for people to avoid using it, to avoid this issue, I would have to make the default date format = `iso`, but that might break existing code, right?

Or could I be misunderstanding and you are just suggesting to remove the `use ``iso`` instead.` message

Reply:
OK ignore what I said above. In a future world we can get rid of this argument altogther, but I suppose for the first deprecation we should make it so that the only valid arguments are None / iso